README.md

```diff
@@ -58,6 +58,7 @@ Command           | Package Id | Author | Description
 `dotnet-sonarscanner` | [dotnet-sonarscanner](https://www.nuget.org/packages/dotnet-sonarscanner) | [@SonarSource](https://github.com/SonarSource) | The SonarScanner for MSBuild is the recommended way to launch a SonarQube or SonarCloud analysis for projects/solutions using dotnet command as build tool.  [GitHub](https://github.com/SonarSource/sonar-scanner-msbuild)
 `dotnet-sshdeploy` | [dotnet-sshdeploy](https://www.nuget.org/packages/dotnet-sshdeploy) | [@Unosquare](https://github.com/unosquare) | A dotnet CLI command that enables quick deployments over SSH.  [GitHub](https://github.com/unosquare/sshdeploy)
 `dotnet-symbol` | [dotnet-symbol](https://www.nuget.org/packages/dotnet-symbol/) | [@microsoft](https://github.com/microsoft) | Symbols download utility.
+`dotnet-t4` | [dotnet-t4](https://www.nuget.org/packages/dotnet-t4/) | [@mhutch](https://github.com/mhutch) | T4 text template processor. [GitHub](https://github.com/mono/t4)
```

Comment:
If I'm not mistaken @mhutch the command is actually `t4`. That's what should be in the first column, and it also changes ordering (which is by command, not package name).

Reply:
Ah, good catch